bskyembed/src/main.tsx

```diff
@@ -1,9 +1,88 @@
 import './index.css'
 
+import {AppBskyFeedDefs, BskyAgent} from '@atproto/api'
 import {h, render} from 'preact'
 
-import {App} from './app'
+import logo from '../assets/logo.svg'
+import {Container} from './container'
+import {Link} from './link'
+import {Post} from './post'
+import {getRkey} from './utils'
 
 const root = document.getElementById('app')
 if (!root) throw new Error('No root element')
-render(<App />, root)
+
+const searchParams = new URLSearchParams(window.location.search)
+
+const agent = new BskyAgent({
+  service: searchParams.get('service') || 'https://public.api.bsky.app',
```

Comment:
I don't think we can do an open proxy like this

Reply:
even with the xrpc constraints we'd probably be smart to just hardcode this to our service

Reply:
Oh my bad, this seems to be client side. In that case, not a concern

Reply:
Right. Ignore me.

Reply:
Oh - just removed it. Should I revert?